README.md

```diff
@@ -1,14 +1,18 @@
 # MoveIt! Tutorials
 
-This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/indigo/api/moveit_tutorials/html/
+This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
```

Comment:
We are currently not "building" this package like a normal ROS package, only running doc jobs on it. https://github.com/ros/rosdistro/pull/12976/files